Terraform/mainnet/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+# This is a set of values to be referenced as variables in the main code
+
+size                    = "s-4vcpu-8gb"
+image                   = "docker-20-04"
+region                  = "fra1"
+source_addresses        = ["0.0.0.0/0", "::/0"]
+destination_addresses   = ["0.0.0.0/0", "::/0"]
+backups                 = "false"
```

Comment:
Not sure it ever makes sense to set `backups` to `true` so we can probably just delete this variable.